Maplike and Setlike objects in Map and Set (#27160)
mdn/content: files/en-us/web/api/xranchorset/index.md

{{APIRef("WebXR Device API")}}{{SeeCompatTable}}

> The **`XRAnchorSet`** interface exposes a collection of anchors. It is returned by {{domxref("XRFrame.trackedAnchors")}} and is a [`Set`-like object](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#set-like_objects) object.

Reviewer: ```suggestion
The **`XRAnchorSet`** interface exposes a collection of anchors. Its instances are returned by {{domxref("XRFrame.trackedAnchors")}} and are [`Set`-like objects](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#set-like_objects).
```
